sql-plugin/src/main/321+-db/scala/com/nvidia/spark/rapids/shims/FileSourceScanExecMeta.scala

```diff
@@ -89,6 +99,24 @@ class FileSourceScanExecMeta(plan: FileSourceScanExec,
     }
   }
 
+  // Support dataFilters in Dynamic File Pruning
+  private lazy val dataFilters = {
+    wrapped.dataFilters.map { filter =>
```

Comment:
this looks like exact same code as above for partitionFilters, can we make a function for it?